src/data2img.py

```diff
@@ -6,6 +6,9 @@
 IMG_SIZE = 48
 CSV_FILE_NAME = '../data/head.csv'
 DATA_OUTPUT_DIR = '../data/img'
+DATA_TRAINING_OUTPUT_DIR = '../data/img/train'
+DATA_TEST_OUTPUT_DIR = '../data/img//test'
+TRAIN_END_POINT = 28708
```

Comment:
this train_end_point is now defined in four different places ~~

also we maybe should be defined by a ratio rather than a fixed number